Reivew and suggested changes (#3)
CTF-handbooks/map-tester-handbook: HANDBOOK.md

![Getting the files Example](images/pr-forked-repo-code-example.png)

- Extract the ZIP file and select the particular map folder that has been added by the author of the PR within the extracted folder. In the example PR mentioned, the folder is `sewer_village`.

> - After successfully identifying the map folder, copy and paste it to `<YOUR MINETEST FOLDER>/games/capturetheflag/mods/ctf/ctf_map/maps/` on your computer. You can go to your Minetest folder and paste the following into your file app's directory to navigate there quickly:
>   ```
>   /games/capturetheflag/mods/ctf/ctf_map/maps
>   ```

Reviewer: Here on Nautilus (and perhaps some other file managers as well) it doesn't behave that way. That might be the case on Windows, but other file managers might behave differently as in my case. I'd suggest to be safe, this line can be removed. Another way to include a similar snippet would be to ask them to search just the `maps` folder within their file manager.

So along the lines of:
```suggestion
- After successfully identifying the map folder, copy and paste it to `<YOUR MINETEST FOLDER>/games/capturetheflag/mods/ctf/ctf_map/maps/` on your computer. You can go to your Minetest folder and search for the `maps` folder in your file manager's search to navigate there quickly.
```
Please have a look, and see as you find fit.
